bundles/org.jkiss.utils/src/org/jkiss/utils/CommonUtils.java

```diff
@@ -933,11 +933,13 @@ public static String escapeHtml(@Nullable String text) {
         }
         return text
             .replace("&", "&amp;")
+            .replace(" ", "&nbsp")
```

Comment:
Is the semicolon optional here?

Reply:
Although some named characters in HTML don't have the semicolon, the `&emsp;` is an exception.

Please see https://html.spec.whatwg.org/multipage/named-characters.html